src/html-behavior.js

```diff
@@ -168,6 +168,19 @@ export class HtmlBehaviorResource {
   register(registry: ViewResources, name?: string): void {
     if (this.attributeName !== null) {
       registry.registerAttribute(name || this.attributeName, this, this.attributeName);
+
+      if (Array.isArray(this.target.aliases)) {
+        this.target
+          .aliases
+          .filter((a, pos, arr) =>
+             a !== this.attributeName &&
+             typeof a === 'string' &&
+             a.length > 0 &&
+             arr.indexOf(a) === pos)
```

Comment:
It seems that `registerAttribute` already checked for empty attribute name and if there's a duplicate.
I think the filter part can/should be removed all together

https://github.com/aurelia/templating/blob/master/src/view-resources.js#L6